src/helper_addressbook.py

```diff
@@ -0,0 +1,16 @@
+"""
+Insert value into addressbook
+"""
+
+from bmconfigparser import BMConfigParser
+from helper_sql import sqlExecute
+
+
+def insert(address, label):
+    """perform insert into addressbook"""
+
+    is_valid = False
+    if address not in BMConfigParser().addresses():
+        sqlExecute('''INSERT INTO addressbook VALUES (?,?)''', label, address)
```

Comment:
you need to verify the reqturn value of `sqlExecute`, which is the number of modified rows